MIA FTWD Jaime Jacobo and Rachel Hofer

JaimeJacobo opened the pull request.

MarcosPalaciosIII commented on starter-code/basic-algorithms.js:
```diff
@@ -1,7 +1,67 @@
+  blah3 = blah3 + (userString[i])
+}
+for(let i = userString.length-1; i > -1; i--) {
+  blah4 = blah4 + (userString[i])
```
another method for doing this is ``` blah4.split(' ').reverse().join(' ')```

MarcosPalaciosIII commented:
Great job on the basic algorithm lab. There are some other methods you could have used other than doing for loops for majority of the challenges, but with time you will learn them. Good use of for loops. 

Keep up the great work.

JaimeJacobo commented:
Hey Marcos!

Thanks for the feedback. I have updated both
exercises ('lab-javascript-functions-and-arrays' and
'lab-javascript-basic-algorithms') and I have added them to Github. So at
this point I have in my Github the 3 exercises completed of Javascript.

This weekend I will finish the CSS exercises.

See you tomorrow!
Jaime

On Wed, Oct 24, 2018 at 11:21 AM MarcosPalaciosIII <notifications@github.com>
wrote:

> Great job on the basic algorithm lab. There are some other methods you
> could have used other than doing for loops for majority of the challenges,
> but with time you will learn them. Good use of for loops.
>
> Keep up the great work.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ironhack-labs/lab-javascript-basic-algorithms/pull/255#issuecomment-432702123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Al_e9nGHSl3rzS9z-PTX7rBT0B0MzF5Gks5uoIVqgaJpZM4X0EW4>
> .
>
